Kitodo/src/main/webapp/pages/processes.xhtml

```diff
@@ -16,14 +16,18 @@
         xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
         xmlns:h="http://xmlns.jcp.org/jsf/html"
         xmlns:f="http://xmlns.jcp.org/jsf/core"
-        xmlns:p="http://primefaces.org/ui">
+        xmlns:p="http://primefaces.org/ui"
+        xmlns:o="http://omnifaces.org/ui">
     <f:metadata>
         <f:viewParam name="tabIndex"/>
         <!--@elvariable id="tabIndex" type="java.lang.Integer"-->
         <f:viewAction action="#{ProcessForm.setActiveTabIndexFromTemplate(tabIndex)}" />
         <!--@elvariable id="keepPagination" type="java.lang.String"-->
         <f:viewParam name="keepPagination"/>
         <f:viewAction action="#{ProcessForm.resetPaginator(keepPagination)}"/>
+        <!--@elvariable id="projecttitle" type="java.lang.String"-->
+        <f:viewParam name="projecttitle"/>
+        <o:viewAction action="#{ProcessForm.setFilter('project:' += projecttitle)}" if="#{not empty projecttitle}"/>
```

Comment:
why do you use the omniface specific viewaction?

Reply:
It offers conditional bean action execution via the optional `if` attribute.